Cloud connected care: update slack interaction (#20959)
pingcap/docs: tidb-cloud/connected-slack-ticket-interaction.md

Reviewer: Preview: https://pingcap-docsite-preview-git-preview-cl-ff6ab9-oreoxmts-projects.vercel.app/tidbcloud/connected-slack-ticket-interaction/

# Interact with Support Tickets via Slack

For customers subscribed to the **Premium** [support plan](/tidb-cloud/connected-care-detail.md), TiDB Cloud provides a ticket bot called **PingCAP Support Bot** in [Slack](https://slack.com/), to support more comprehensive interaction and management of support tickets.

> **Note:**
>
> The ticket support feature for Slack is available upon request. If you are interested in trying this feature, contact TiDB Cloud support at <a href="mailto:support@pingcap.com">support@pingcap.com</a> or reach out to your Technical Account Manager (TAM).

> Use **PingCAP Support Bot** to raise a support ticket in Slack.

Reviewer: ```suggestion
You can use **PingCAP Support Bot** to create a support ticket in Slack:
```

[slack-ticket-interaction-creation](/media/tidb-cloud/connected-slack-ticket-interaction-creation.gif)

> Reply to support tickets directly in Slack.

Reviewer: ```suggestion
Then, you can reply to the support ticket directly in Slack:
```

> [slack-ticket-interaction-reply](/media/tidb-cloud/connected-slack-ticket-interaction-reply.gif)

Reviewer: ```suggestion
![Reply to a support ticket in Slack](/media/tidb-cloud/connected-slack-ticket-interaction-reply.gif)
```

## Interact with support tickets

> In the Slack support channel, you only need to mention **PingCAP Support Bot** and describe the problem in a message. Then, the Bot will send you a message with a **Raise request** button.

Reviewer: ```suggestion
In the Slack support channel, you only need to mention **PingCAP Support Bot** and describe the problem in a message. Then, the bot will send you a message with a **Raise request** button.
```
